File: big_data_learning/spark-learning/com/shawn/study/spark/core/builder/create-rdd-by-memory.py
from pyspark import SparkConf, SparkContext

if __name__ == '__main__':
    conf = SparkConf().setAppName(value="create-rdd-by-memory").setMaster(value="local[*]")
    context = SparkContext(conf=conf)
    rdd = context.parallelize([1, 2, 3])
    print(rdd.collect())
    context.stop()

File: big_data_learning/spark-scala-java-learning/src/main/java/com/shawn/study/big/data/spark/java/core/rdd/builder/CreateRddByMemory.java
package com.shawn.study.big.data.spark.java.core.rdd.builder;

import java.util.Arrays;
import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaRDD;
import org.apache.spark.api.java.JavaSparkContext;

public class CreateRddByMemory {

  public static void main(String[] args) {
    SparkConf conf = new SparkConf().setAppName("create-rdd-by-memory").setMaster("local[*]");
    JavaSparkContext context = new JavaSparkContext(conf);
    JavaRDD<String> rdd = context.parallelize(Arrays.asList("1", "2", "3"));
    rdd.collect().forEach(System.out::println);
    context.close();
  }
}
